examples/ChatQnA/deploy/gaudi.md

```diff
@@ -297,28 +297,28 @@ Set the necessary environment variables to setup the use case case
 
 ### Dataprep
 
-    export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"
-    export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_file"
-    export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_file"
+export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"
+export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_file"
+export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_file"
 
-  ### VectorDB
+### VectorDB
 
-    export REDIS_URL="redis://${host_ip}:6379"
-    export INDEX_NAME="rag-redis"
+export REDIS_URL="redis://${host_ip}:6379"

```

Comment:
Removing the indent lost the code-block formatting. Put the four-space indent back.